<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>calvpattern</class>
 <widget class="impgenmaint" name="calvpattern">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>673</width>
    <height>325</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="Label21">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>(Number of calvings/lambings per period)</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QTableView" name="AdvListView4">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>539</width>
         <height>0</height>
        </size>
       </property>
       <property name="toolTip">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="BitBtn1">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="toolTip">
        <string>Save the current record</string>
       </property>
       <property name="text">
        <string>Apply calendar</string>
       </property>
       <property name="icon">
        <iconset resource="../images.qrc">
         <normaloff>:/images/apply.png</normaloff>:/images/apply.png</iconset>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="BitBtn5">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="toolTip">
        <string>Restore the current record</string>
       </property>
       <property name="text">
        <string>Cancel calendar</string>
       </property>
       <property name="icon">
        <iconset resource="../images.qrc">
         <normaloff>:/images/cancel.png</normaloff>:/images/cancel.png</iconset>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="BitBtn13">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="toolTip">
        <string/>
       </property>
       <property name="text">
        <string>Close</string>
       </property>
       <property name="icon">
        <iconset resource="../images.qrc">
         <normaloff>:/images/closeform.png</normaloff>:/images/closeform.png</iconset>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>impgenmaint</class>
   <extends>QWidget</extends>
   <header>impgenmaint.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../images.qrc"/>
 </resources>
 <connections/>
</ui>
